### PR TITLE
Adds seeding script for MySQL

### PR DESCRIPTION
**Pull request by bellcd**
Please ignore the Cassandra work on the pull request, it's still [wip]

### CODE REVIEW

File: database/mysql-schema.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  FOREIGN KEY (org_id)
+    REFERENCES org (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE,
+  person_id INT,
+  FOREIGN KEY (person_id)
+    REFERENCES person (id)
+    ON DELETE CASCADE
+    ON UPDATE CASCADE,
+  founder BOOLEAN,
```
Comment (RomanEmmons) [inline database/mysql-schema.sql]: I thought sql didn't have a BOOLEAN data type?  I could be wrong but I though BIT was how you set true/false values.

File: database/mysql-schema.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  series_description VARCHAR(255),
+  day_of_week VARCHAR(255),
+  series_interval INT
+);
+
+CREATE TABLE person (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  first_name VARCHAR(255),
+  last_name VARCHAR(255),
+  -- this was a holdover from the legacy database (MongoDB), where the identifier for each person was a string like `m0` `m145` `m487` etc...
```
Comment (RomanEmmons) [inline database/mysql-schema.sql]: I ran into the same thing with the service I'm working on. I changed it to an integer... prob destroyed the front end when I did.

Comment (bellcd) [reply database/mysql-schema.sql]: ooof

File: database/mysql-schema.sql (hunk 1 above)
Comment (bellcd) [reply database/mysql-schema.sql]: [BOOLEAN](https://www.mysqltutorial.org/mysql-boolean/) seems to be working for me... They are stored as 0s or 1s, which is neat.

**Pull request merged**